enigma-swing/src/main/java/org/quiltmc/enigma/gui/config/Config.java

```diff
@@ -69,20 +94,25 @@ public final class Config extends ReflectiveConfig {
 	public final DevSection development = new DevSection();
 
 	/**
-	 * The look and feel stored in the config: do not use this unless setting! Use {@link #activeLookAndFeel} instead,
+	 * The look and feel stored in the config: do not use this unless setting! Use {@link #activeThemeProperties} instead,
 	 * since look and feel is final once loaded.
 	 */
-	public final TrackedValue<LookAndFeel> lookAndFeel = this.value(LookAndFeel.DEFAULT);
+	public final TrackedValue<ThemeProperties> theme = this.value(ThemeProperties.DEFAULT);
 	/**
 	 * Look and feel is not modifiable at runtime. I have tried and failed multiple times to get this running.
 	 */
-	public static LookAndFeel activeLookAndFeel;
-
-	public final Theme defaultTheme = new Theme(LookAndFeel.DEFAULT);
-	public final Theme darculaTheme = new Theme(LookAndFeel.DARCULA);
-	public final Theme metalTheme = new Theme(LookAndFeel.METAL);
-	public final Theme systemTheme = new Theme(LookAndFeel.SYSTEM);
-	public final Theme noneTheme = new Theme(LookAndFeel.NONE);
+	public static ThemeProperties activeThemeProperties;
+
+	public final Theme defaultTheme = new Theme(ThemeProperties.DEFAULT);
```

Comment:
would you mind moving these to a new file? if we're making breaking changes i'd like to declutter the main config file a bit

Reply:
that could be a good solution actually, idk if qconf supports subdirs though